### PR TITLE
LIB-733: Default state of fancy code snippet is not hover state

### PR DESCRIPTION
**Pull request by oriooctopus**

### CODE REVIEW

File: src/theme/Tabs/index.js [hunk 1]
```diff
@@ -74,6 +76,15 @@ function Tabs(props) {
     }
   };
 
+  /*
+   * Even though a tab is selected on render, we don't want to show the
+   * selected/hover state on the tab headers until subsequent renders
+   */
+  const onChange = cb => {
```
Comment (jjwill) [inline src/theme/Tabs/index.js]: nit: change to `callback` or `handler`

**Review by jjwill — COMMENTED** [review]
Can you spend 30min trying to get this to only set the state after render (does setting state cause another render?) so we can avoid calling this for every event?

**Comment by oriooctopus** [comment]
@jjwill most effective thing to do would just be to only run setState if hasHadChange is false, that way it only runs one time

**Review by jjwill — APPROVED** [review]
LGTM

**Pull request merged**